contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_meta_relocate_integration.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class MetaRelocateIntegrationTest(PantsRunIntegrationTest):
+  def test_meta_relocate(self):
+
+    pre_dependees_run = self.run_pants(['dependees',
+      'testprojects/tests/java/org/pantsbuild/testproject/buildrefactor:X'])
+
+    self.run_pants(['meta-relocate',
+      '--to=testprojects/tests/java/org/pantsbuild/testproject/buildrefactor/x:X',
+      'testprojects/tests/java/org/pantsbuild/testproject/buildrefactor:X'])
+
```

Comment:
could you make an assertion here to make sure the references to `testprojects/tests/java/org/pantsbuild/testproject/buildrefactor:X` have been updated to `testprojects/tests/java/org/pantsbuild/testproject/buildrefactor/x:X`?